tflint-ruleset-blueprint/rules/testdata/terraform_required_version/multiple-invalid/main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
```

Comment:
This is likely handled by go-version and we are unlikely to use any way but could we have a `~>` operator test?

Reply:
Good call - Yeah, I switch from `mod/semver` to `go-version` for this exact type of reason.